docs/source/x_ext.rst

```diff
@@ -557,32 +557,42 @@ Commit interface
-   ``commit_kill`` signal for instructions that it did not accept. A |processor| can signal either ``commit_kill`` = 0 or ``commit_kill`` = 1 for non-accepted instructions.
-
 :numref:`Commit packet type` describes the ``x_commit_t`` type.
 
 .. table:: Commit packet type
   :name: Commit packet type
 
+  +--------------------+------------------------+------------------------------------------------------------------------------------------------------------------------------+
+  | **Signal**         | **Type**               | **Description**                                                                                                              |
   +--------------------+------------------------+------------------------------------------------------------------------------------------------------------------------------+
   | ``id``             | :ref:`id_t <id>`       | Identification of the offloaded instruction. Valid when ``commit_valid`` is 1.                                               |
   +--------------------+------------------------+------------------------------------------------------------------------------------------------------------------------------+
   | ``commit_kill``    | logic                  | Shall an offloaded instruction be killed? If ``commit_valid`` is 1 and ``commit_kill`` is 0, then the core guarantees        |
   |                    |                        | that the offloaded instruction (``id``) is no longer speculative, will not get killed (e.g. due to misspeculation or an      |
   |                    |                        | exception in a preceding instruction), and is allowed to be committed. If ``commit_valid`` is 1 and ``commit_kill`` is       |
   |                    |                        | 1, then the offloaded instruction (``id``) shall be killed in the |coprocessor| and the |coprocessor| must guarantee that the|
-  |                    |                        | related instruction does/did not change architectural state.                                                                 |
+  |                    |                        | Shall an offloaded instruction be commited or killed?                                                                        |
+  |                    |                        | If ``commit_valid`` is 1 and ``commit_kill`` is 0,  then the core guarantees that the offloaded instruction (``id``) and any |
+  |                    |                        | older (i.e. preceeding) instructions are no longer speculative, will not get killed (e.g. due to misspeculation or an        |
+  |                    |                        | exception in a preceding instruction), and are allowed to be committed.                                                      |
+  |                    |                        | If ``commit_valid`` is 1 and ``commit_kill`` is 1, then the offloaded instruction (``id``) and any newer (i.e. succeeding)   |
```

Comment:
The words newer/succeeding/older/preceeding are not well defined. They need to get defined relative to the issue_valid/issue_ready handshake as well.